scripts/updateApis.ts

```diff
@@ -21,13 +21,44 @@ if (!process.env.ANYPOINT_USERNAME || !process.env.ANYPOINT_PASSWORD) {
 const OLD_APIS_PATH = path.join(__dirname, '../temp/oldApis');
 const PRODUCTION_API_PATH = path.join(__dirname, '../apis');
 
+/**
+ * Recursively removes all files ending in '-internal.yaml' from a directory and its subdirectories
+ * @param directoryPath - The path to the directory to process
+ */
+function removeInternalOas(directoryPath: string): void {
```

Comment:
Nit: Do we want to cut this function short by expecting a directory to have like 3 depth. If it goes beyond that, we error out? I am thinking of a scenario where someone accidentally created lik 10 nested folder structure, and this func will try to run them all. 😬